command-message/command-message.bal

```diff
@@ -0,0 +1,36 @@
+import ballerina/http;
+
+type Usergroup record {
+    string id;
+    string team_id;
+    boolean is_usergroup;
+    string name;
+    string description;
+    string 'handle;
+    boolean is_external;
+    int date_create;
+    int date_update;
+    int date_delete;
+    string created_by;
+    string? updated_by;
+    string? deleted_by;
+    string user_count;
+};
```

Comment:
Do we need all these fields?

Reply:
+1 to reduce fields.
why is it `Usergroup` not `UserGroup`?

Reply:
updated.